database/__init__.py

```diff
@@ -301,10 +328,15 @@ def login():
             flash('Invalid user or password', 'danger')
             return redirect(url_for('login', next=request.args.get('next')))
         login_user(user)
-        return redirect(request.args.get('next', url_for('index')))
+        return redirect(url_for(request.args.get('next', 'index')))
     return render_template('login.html', title='Login', form=form)
 
 
+@app.route('/config/pars/')
```

Comment:
What is this for?

Reply:
It is used by the PARS front end to configure UI elements. If we drop pars, we can drop this.